frontend/src/static/js/v_authorship.js

```diff
@@ -152,10 +150,10 @@ window.vAuthorship = {
 
     sortFileTypeAlphabetically(unsortedFilesInfoObj) {
       return Object.keys(unsortedFilesInfoObj)
-          .sort()
-          .reduce((acc, key) => ({
-              ...acc, [key]: unsortedFilesInfoObj[key]
-          }), {});
+        .sort()
```

Comment:
hmm... this doesn't seems to be right as its linewrapped

Reply:
i put `MemberExpression: 2` already, gives me the same thing :disappointed:

Reply:
Does the `MemberExpression` even affects it? If its not then probably its other rules.